apps/ui/src/components/Onboarding.vue

```diff
@@ -0,0 +1,63 @@
+<script setup lang="ts">
+const usersStore = useUsersStore();
+const { web3 } = useWeb3();
+const followedSpacesStore = useFollowedSpacesStore();
+
+const user = computed(() => {
+  if (!web3.value.authLoading) {
```

Comment:
```suggestion
  if (!web3.value.authLoading && web3.value.account) {
```

Else this will get triggered even for logged out users